snakesee/parser/line_parser.py

```
@@ -0,0 +1,288 @@
+        value: String to parse.
+        field_name: Name of field for logging.
+
+    Returns:
+        Parsed integer if valid and positive, None otherwise.
+    """
+    try:
+        result = int(value)
+        if result <= 0:
+            logger.debug("Invalid %s: %d (must be positive)", field_name, result)
+            return None
+        return result
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)